jwt.el

```diff
@@ -613,13 +613,58 @@ Specifically it checks that TEST-STRING has
            (doco (car maybe-doc)))
       (funcall callback doco :thing full-name))))
 
+(defvar jwt--annotation-claims
+  '(("nbf" . "Not before")
+    ("iat" . "Issued at")
+    ("exp" . "Expires")))
+
+(defface jwt-annotation
+  '((t
+     (:box
+      (:line-width (1 . 1)
+                   :color nil
+                   :style nil)
+      :inherit (magit-branch-local))))
```

Comment:
This should inherit from a built-in face just in case (maybe?) someone isn't using magit!